precompiles/collective/src/mock.rs

```diff
@@ -198,6 +202,10 @@ impl pallet_treasury::Config for Runtime {
 	>;
 }
 
+parameter_types! {
+	pub MaxProposalWeight: Weight = Weight::from_parts(1_000_000_000, 1_000_000_000);
```

Comment:
Isnt this rather low on the weight side? 1 billion seems quite low for things like XCM